acceptance/testdata/runnerdeploy.yaml

```diff
@@ -6,7 +6,10 @@ spec:
 #  replicas: 1
   template:
     spec:
-      repository: mumoshu/actions-runner-controller-ci
+      repository: ${NAME}
```

Comment:
Why are there envvar-like expressions in this file now? Are they supposed to be expanded by anything? (I imagine if there's any small script that involves e.g. sed to replace these expressions but I couldn't fine one in a quick look

Reply:
This is now called by being piped into a envsubst which expands them and then to kubectl, that way there doesn't have to be an expanded forever changing copy on the disk.